src/pages/Products/components/ProductOperations.js

```diff
@@ -0,0 +1,148 @@
+import { useState } from 'react';
+import { useData } from '../../../context';
+import styles from './ProductOperations.module.css';
+import CloseIcon from '@material-ui/icons/Close';
+
+export function ProductOperations() {
+  const [sortModal, showSortModal] = useState(false);
+
+  const [filterModal, showFilterModal] = useState(false);
+
+  const { showInventoryAll, showFastDelivery, dataDispatch, priceSlider } = useData();
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.sort} onClick={() => showSortModal((sortModal) => !sortModal)}>
+        Sort
+      </div>
+      {sortModal && (
+        <div
+          className={styles.modalCloseContainer}
+          onClick={() => {
+            showSortModal((sortModal) => !sortModal);
+          }}
+        >
+          <div
+            className={`${styles.modal} ${styles.sortModal}`}
+            onClick={(e) => {
+              e.stopPropagation();
+            }}
```

Comment:
hey pranshu, where are you using this code in your deployed app
i am unable to see the `sort` toggle in the deployed app